packages/cli/plugin-storybook/src/index.ts

```diff
@@ -39,6 +40,19 @@ export default (pluginOption: PluginOptions = {}): CliPlugin<ModuleTools> => ({
       return config;
     },
 
+    beforeBuild: async () => {
+      const { fs } = await import('@modern-js/utils');
+      const { STORYBOOK_DIST_DIR_NAME } = await import('./features/constants');
+      const { appDirectory } = api.useAppContext();
+      const storybookDistPath = path.join(
+        appDirectory,
+        'dist',
```

Comment:
We can use`appContext.distDirectory`

Reply:
How is `distDirectory` calculated? It may not have much to do with Module Tools

Reply:
This output path is currently temporary and an api should be designed later.

Reply:
You are right, ignore this comment.